manifest.json

```diff
@@ -15,14 +15,16 @@
     "default_popup": "html/popup.html"
   },
   "background": {
-    "scripts": ["js/utils.js", "js/background.js"]
+    "scripts": [ "js/background.js" ],
+    "service_worker": "js/background.js",
+    "type": "module"
```

Comment:
These changes seem to break the default config from being loaded on initial install of the extension (on Firefox).

I'm not sure if there is a way to make both manifests work (but the rest of the code seems to work, so this is awesome).

What do you think of having two separate manifests for chrome and firefox?

Reply:
Hmm interesting, I don't run into that issue. But yes I agree it'll be less of a headache to just have two manifests.
How about keeping the main manifest.json for firefox and adding another one for chrome (manifest-chrome.json)?

Reply:
Ah, for some reason my nix config was pulling in an old version of Firefox Dev edition. After bumping to 121, the config is loading now. Maybe we can try to see how long a single manifest can work?

Reply:
Either way, this LGTM.

At some point, it would be nice to submit this to the chrome web store for easier installation.

Reply:
Once chrome 121 is released to stable I think we can switch to a single manifest. We can probably submit to the chrome web store at that point as well